zap/zap.gradle.kts

```diff
@@ -201,6 +201,7 @@ val japicmp by tasks.registering(JapicmpTask::class) {
         "org.parosproxy.paros.network.ConnectionParam#getProxyChainSkipName()",
         "org.parosproxy.paros.network.ConnectionParam#setProxyChainSkipName(java.lang.String)",
         "org.parosproxy.paros.view.AbstractFrame#loadIconImages()",
+        "org.parosproxy.paros.control.Control#createAndOpenUntitledDb()",
```

Comment:
This should be in alphabetical order. Other than that this PR looks good so far :)